README.md

```diff
@@ -20,10 +20,35 @@ Clone the repository, make sure Docker is installed, and then:
 
 ```
 pip install -r requirements.txt
-./build_run_test.sh
+python test.py
 ```
 
-After the tests run the container is left up, with the local url where it can be accessed.
+After the tests run the containers used for testing are killed, but one can be brought up again with the following command: 
+    - `docker run -p 8080:80 -e INPUT_JSON_URL=https://raw.githubusercontent.com/refinery-platform/docker_igv_js/master/input_fixtures/good/input.json docker_igv_js`
+
+An `input.json` file can also be constructed and passed to the container as an envvar: `INPUT_JSON`.
+
+    - Create a file locally containing the minimal portions of the JSONSchema required by the [refinery-platform](https://github.com/refinery-platform/refinery-platform)
+      - ```
+        {
+          "node_info": {
+            "<some unique key>": {
+              "file_url": "<url to an IGV-compatible file",
+              "node_solr_info": {
+                "name": "<some descriptor of your file to be a prefix in your track name>"
+              }
+            }
+          },
+          "parameters": [
+            {
+              "name": "Genome Build",
+              "value": "hg19"
+            }
+          ]
+        }
+        ```
+    - `docker run -p 8080:80 -e INPUT_JSON="$(cat <file name containing the above json>)" docker_igv_js`
```

Comment:
I'd prefer that this be done with a multi-line quote, and just use plausible fake values so it can run after copy and pasting. Something like:
```
docker run -p 8080:80 -e INPUT_JSON='
        {
          "node_info": {
            "id-1": {
              "file_url": "https://github.com/igvteam/igv/raw/master/test/data/bam/NA12878.SLX.sample.bam",
              "node_solr_info": {"name": "track-name"}
            }
          },
          "parameters": [
            {"name": "Genome Build", "value": "hg19"}
          ]
        }
' gehlenborglab/docker_igv_js
```